docs/source/notes/create_gnn.rst

```diff
@@ -103,11 +103,11 @@ Here, we first add self-loops to our edge indices using the :meth:`torch_geometr
 The normalization coefficients are derived by the node degrees :math:`\deg(i)` for each node :math:`i` which gets transformed to :math:`1/(\sqrt{\deg(i)} \cdot \sqrt{\deg(j)})` for each edge :math:`(j,i) \in \mathcal{E}`.
 The result is saved in the tensor :obj:`norm` of shape :obj:`[num_edges, ]` (step 3).
 
-We then proceed to call :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.propagate`, which internally calls the :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.message`, :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.aggregate` and :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.update` functions.
-As additional arguments for message propagation, we pass the node embeddings :obj:`x` and the normalization coefficients :obj:`norm`.
+We then call :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.propagate`, which internally calls :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.message`, :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.aggregate` and :meth:`~torch_geometric.nn.conv.message_passing.MessagePassing.update`.
+We pass the node embeddings :obj:`x` and the normalization coefficients :obj:`norm` as additional arguments for message propagation.
```

Comment:
This change seems minor

Reply:
The change to line 107 fixes the "dangling modifier" grammatical error. The phrase "as additional arguments for message propagation" describes the embeddings and coefficients, but the original sentence is structured such that the phrase incorrectly describes "we."